web/src/js/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import * as modalAction from '../../ducks/ui/modal'
+import ModalList from './ModalList'
+
+class PureModal extends Component {
+
+    constructor(props, context) {
+        super(props, context)
+    }
+
+    render() {
+        const { activeModal, hideModal } = this.props
+        const ActiveModal = _.find(ModalList, m => m.name === activeModal )
```

Comment:
Should we remove `<div className="modal-backdrop fade in"></div>` to omit the dark background  or put it into every modal component in the ModalList?